frontend/packages/kubevirt-plugin/src/k8s/requests/cdi-upload/cdi-upload-requests.ts

```diff
@@ -14,11 +18,7 @@ const UPLOAD_STATES = {
   READY: 'UploadReady',
 };
 
-const waitForUploadReady = async (
-  dataVolume: K8sResourceKind,
-  k8sGet: (kind: K8sKind, name: string, namespace: string) => Promise<K8sResourceKind>,
-  counter: number = 30,
-) => {
+const waitForUploadReady = async (dataVolume: K8sResourceKind, counter: number = 30) => {
```

Comment:
@glekner I noticed EnhancedK8sMethods are used in cdi upload even though their functionality is never triggered so I reverted back to normal K8s* methods. Let me know if I missed something and they are indeed needed.